build/.mxchip-ci.yml

```diff
@@ -1,28 +1,58 @@
-# Copyright (c) Microsoft. All rights reserved.
-# Licensed under the MIT license. See LICENSE file in the project root for full license information.
 name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
-resources:
-- repo: self
-  clean: true
-phases:
-- phase: iotz
-  queue:
-    name: devicelab-01
+
+jobs:
+- job: Cross_Compile
+  pool:
+    vmImage: ubuntu-16.04
   steps:
   - script: |
-      sudo apt-get update && apt-get install -y \
-      curl \
-      git \
-      python-software-properties \
-      build-essential \
-      pkg-config
-      sudo curl -sL https://deb.nodesource.com/setup_6.x | bash -
-      sudo apt-get install -y nodejs
+      . build_all/mbed5/setup.sh
```

Comment:
can you point to a build that verifies that this yaml works?

Reply:
https://dev.azure.com/azure-iot-sdks/azure-iot-sdks/_build/results?buildId=18725